docs/api-docs/storefront/widgets/widgets-overview.md

```diff
@@ -348,7 +348,7 @@ Widgets are rendered on the storefront as a data tag in the HTML.
 | Widget templates | Widget templates are Handlebars-enabled HTML templates which define the widget’s structure on a page. These templates can include conditional logic as well as looping. *There is a limit of `1,000` total custom widget templates per store. This does not include templates pre-provided by BigCommerce.* |
-| Placements | Placements are the records to track which widget appears on which page, and in what order.  Currently, placements can only exist on the following pages: </br>* pages/blog-post </br> * pages/blog</br>* pages/brand</br>* pages/brands</br>* pages/cart</br>* pages/category</br>* pages/home</br>* pages/page</br>* pages/product</br>* pages/search</br> *There is a limit of 75 placements per template file and `10,000` total placements per store.*|
+| Placements | Placements determine the region where you place widgets and in what order. Currently, placements can only exist on the following pages:</br>* pages/account/add-address </br>* pages/account/add-return </br>* pages/account/add-wishlist </br>* pages/account/addresses </br>* pages/account/download-item </br>* pages/account/edit </br>* pages/account/inbox </br>* pages/account/orders/all </br>* pages/account/orders/completed </br>* pages/account/orders/details </br>* pages/account/orders/invoice </br>* pages/account/recent-items </br>* pages/account/return-saved </br>* pages/account/returns </br>* pages/account/wishlist-details </br>* pages/account/wishlists </br>* pages/auth/account-created </br>* pages/auth/create-account </br>* pages/auth/forgot-password </br>* pages/auth/login </br>* pages/auth/new-password </br>* pages/blog </br>* pages/blog-post </br>* pages/brand </br>* pages/brands </br>* pages/cart </br>* pages/category </br>* pages/compare </br>* pages/contact-us </br>* pages/errors/403 </br>* pages/errors/404 </br>* pages/errors/generic </br>* pages/gift-certificate/balance </br>* pages/gift-certificate/purchase </br>* pages/gift-certificate/redeem </br>* pages/home </br>* pages/order-confirmation </br>* pages/page </br>* pages/product </br>* pages/search </br>* pages/sitemap </br>* pages/subscribed </br>* pages/unavailable/hibernation </br>* pages/unavailable/maintenance </br>* pages/unsubscribe </br> *There is a limit of 75 placements per template file and `10,000` total placements per store.*|
 | Regions | Regions are specific spots in a Stencil template file where you can place widgets. Regions are defined at the theme file level using the following syntax: `{{{region name="..."}}}`. Many widgets can reside within a given region, and these widgets can have an assigned sort order. |
 | Widget configuration | This is a JSON payload that contains data used when rendering the widget. Each widget has a configuration, and there is a 64kb limit on the size of the JSON. The widget configuration must be valid JSON, but we don’t enforce any additional configuration structure requirements. |
```

Comment:
Regions are specific spots in a Stencil template file where you can place widgets. Regions are defined at the theme file level using the following syntax: `{{{region name="..."}}}`. Many widgets can reside within a given region, and these widgets can have an assigned sort order. |
| Widget configuration | This is a JSON payload that contains data used when rendering the widget. Each widget has a configuration, and there is a 64kb limit on the JSON size. The widget configuration must be valid JSON, but we don’t enforce any additional configuration structure requirements. |